content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -21,16 +21,21 @@ DVC makes it easy to track [metrics](/doc/command-reference/metrics), update
 <admon type="tip">
 
 All of the above can be combined into [experiments] to run and compare many
-iterations of your ML project.
+iterations of your machine learning project.
 
 [experiments]: /doc/start/experiments
 
 </admon>
 
 ## Collecting metrics
 
-First, let's see the mechanism to capture values for these ML attributes. Add
-and run a final evaluation stage to our [earlier pipeline]:
+Collecting and visualizing metrics is crucial for machine learning
```

Comment:
if we change this, it should be the first sentence in the whole section.

Reply:
It is the first sentence indeed

> "values for these ML attributes." - This is not good imo. confusing. It's not clear what those "ML attributes" are (it's a first sentence)

Reply:
The first one is `DVC makes it easy to track ...` (by section I mean the page, sorry for the confusion).

Reply:
Got it. Moving and adjusting 👍 
Note: this only works with some additional copywriting changes that I believe are not necessary but positive overall. hope you'll agree